### ISSUE TITLE
[Word Mobile]: NVDA is not announcing anything about the author name and time of comment.

### ISSUE THREAD

**Issue by WiproPSAccTest** [nvaccess/nvda#8283]
### Steps to reproduce:
1. Open the sample doc 
2. Navigate to the document edit area and select some text.
3. Navigate to insert tab and insert comment
4. Observe that NVDA is not announcing anything about the author and time of comment.
5. If you reply to the comment then also NVDA is not announcing the author name.
 
### Expected behavior:
NVDA should announce author name as well as time of comment.

### Actual behavior:
NVDA is not announcing anything about the author who has commented and when

### System configuration:
NVDA version: 2018.1.1
App Name: Word
NVDA Installed or portable: Installed
**Other information:**
Windows version:
> Example: Windows 10 Version 1803 Build 17134.48
Name and version of other software in use when reproducing the issue:

### Other questions:
Does the issue still occur after restarting your PC?
Yes
Have you tried any other versions of NVDA?
No
> Please list them and the result

**Comment by Adriani90** [comment]
There is no example document uploaded to this issue. Moreover, this is a broader discussion already covered in #6155 and #4336 
I am closing this issue for now.
